socorro/processor/breakpad_transform_rules.py

```diff
@@ -374,6 +380,22 @@ def _execute_external_process(self, command_line, processor_meta):
         return external_command_output, return_code
 
     #--------------------------------------------------------------------------
+    @staticmethod
+    def dot_save(a_mapping, key, value):
```

Comment:
I feel like I've seen this solution before. Like somewhere in configman around dotdict. I could be wrong but it seems strange to need a whole new staticmethod for something that is only used twice.